types.d.ts

```diff
@@ -0,0 +1 @@
+export * from './out/types'
```

Comment:
I'd rather this file wasn't necessary...

Reply:
Can you elaborate? Importing types from `grammy/types` on Node does not work for me without this file.

Reply:
https://github.com/colinhacks/zod/issues/805

Reply:
What do you mean? A cryptic link is not a good elaboration, are you somehow suggesting this can be fixed in `package.json` or something?

Reply:
Indeed:

```jsonc
  // @filename: package.json
  "exports": {
    ".": {
      "require": "./out/mod.js",
      "types": "./out/mod.d.ts"
    },
    "./types": {
      "types": "./out/types.d.ts"
    },
 }
```

Reply:
```suggestion
export * from './out/types.node.d.ts'
```

Reply:
> Indeed:
> 
> ```js
>   // @filename: package.json
>   "exports": {
>     ".": {
>       "require": "./out/mod.js",
>       "types": "./out/mod.d.ts"
>     },
>     "./types": {
>       "types": "./out/types.node.d.ts"
>     },
>   }
> ```

This was contained in the first commit of this PQ. Why would I have removed it? 🤔